packages/instant/src/components/scaling_input.tsx

```diff
@@ -0,0 +1,153 @@
+import * as _ from 'lodash';
```

Comment:
Nice work on this component -- although it took a little bit of time for me to mentally parse all the logic, you did a good job separating out functionality into small functions that have proper scopes and names 💯